app/src/main/res/values/strings.xml

```diff
@@ -61,6 +61,7 @@
   <string name="lessons">Lessons</string>
   <string name="practice">Practice</string>
   <string name="revision">Revision</string>
+  <string name="administrator_controls_label">Administrator Controls</string>
```

Comment:
Rename this to `administrator_controls_title`

Reply:
Renamed administrator_controls_label to administrator_controls_title: https://github.com/oppia/oppia-android/pull/3152/commits/3ee6278ea4eb85cd5f8ebae54ba6e1125b9bd362